guides/source/active_record_validations.md

```diff
@@ -953,7 +953,7 @@ should happen, an `Array` can be used. Moreover, you can apply both `:if` and
 ```ruby
 class Computer < ApplicationRecord
   validates :mouse, presence: true,
-                    if: ["market.retail?", :desktop?],
+                    if: :desktop?,
```

Comment:
> an `Array` can be used

We should probably change the condition rather than remove it.. otherwise this example loses half its value.